<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="carId">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nowTime">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="inButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>370</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>进入</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>294</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>车辆号码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前时间</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="room">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>210</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>容量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="roomButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>370</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>离开</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="messageLine">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>301</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>进出状况统计</string>
   </property>
  </widget>
  <widget class="QLabel" name="parkspace">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>300</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>停车场内车辆</string>
   </property>
  </widget>
  <widget class="QLabel" name="parkspace2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>240</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>便道内车辆</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
